mentat/code_context.py

```diff
@@ -149,7 +150,7 @@ async def get_code_message(
         )
 
         tokens_used = prompt_tokens + meta_tokens + include_files_tokens
-        if not is_context_sufficient(tokens_used):
+        if context_guard and not is_context_sufficient(tokens_used):
```

Comment:
`is_context_sufficient` should probably be renamed, it sounds like it's asking if context is enough, not checking that it doesn't exceed max.

Looks like the only places it's called, the calling function just raises ReturnToUser if true? maybe we can move the `raise ReturnToUser` to it and rename to `raise_if_context_exceeds_max`

and we can rename `context_guard` to `suppress_context_limit_check` or something else more clear?